subsys/bluetooth/audio/tbs.c

```diff
@@ -2430,7 +2430,6 @@ void bt_tbs_register_cb(struct bt_tbs_cb *cbs)
 	tbs_cbs = cbs;
 }
 
-#if defined(CONFIG_BT_TBS_LOG_LEVEL_DBG)
```

Comment:
the linker will remove this, right @Thalley ?

Reply:
If it won't be used, yes. 

The main point of the guard was to ensure that it wasn't available when debug wasn't enabled. 

The guard can be removed from the header file as you've done (it should in any case). 

We should still be able to guard the implementation with `#if defined(CONFIG_BT_TBS_LOG_LEVEL_DBG)` - Doesn't that work?